README.md

```diff
@@ -170,6 +170,20 @@ final ListenableFuture<B> future = getFuture();
 FuturesExtra.addFailureCallback(future, e -> e.printStackTrace());
 ```
 
+#### Waiting for multiple futures
+
+Sometimes you want to depend on a single future, but for timing reasons it's useful to
+wait for other futures to complete. You can use this method for that:
+```java
+final ListenableFuture<String> future = Futures.immediateFuture("hello");
+final ListenableFuture<String> other1 = Futures.immediateFuture("other1");
+final ListenableFuture<String> other2 = Futures.immediateFuture("other2");
+final ListenableFuture<String> result = FuturesExtra.waitFor(future, other1, other2);
```

Comment:
`waitFor` doesn't really convey what it does to me, but I think it's hard to come up with a better name. It's something like 'returnResultOfFirstWhenAllDone'. Maybe `waitAndReturnFirst` ?